src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -262,7 +271,7 @@ private bool IsValidProjectFilePath(string path)
         /// <returns>Returns true if the path exists and is a sln file type.</returns>
         private bool IsValidSlnFilePath(string path)
         {
-            return File.Exists(path) && Path.GetExtension(path).EndsWith("sln");
+            return File.Exists(path) && (Path.GetExtension(path) == ".sln" || Path.GetExtension(path) == ".slnx");
```

Comment:
or slnf?

Reply:
I think this does not currently support solution filter files

Reply:
I agree, but I guess my question is: why not?

Reply:
I think we could wait and see if/when vs-solutionpersistence implements support for slnf files before trying to parse them on our own